libclc/CMakeLists.txt

```diff
@@ -345,16 +338,8 @@ foreach( t ${LIBCLC_TARGETS_TO_BUILD} )
     set( DARCH ${ARCH} )
   endif()
 
-  set(IS_NATIVE_CPU_ARCH FALSE)
-  if( ARCH IN_LIST NATIVECPU_SUPPORTED_ARCH )
-    set(IS_NATIVE_CPU_ARCH TRUE)
-  endif()
-
-  if( IS_NATIVE_CPU_ARCH AND OS STREQUAL linux)
-    LIST( APPEND dirs native_cpu-unknown-linux )
-  elseif( IS_NATIVE_CPU_ARCH AND NOT OS STREQUAL linux )
-    message(WARNING "libclc is being built for an unsupported ARCH/OS"
-      " configuration, some SYCL programs may fail to build.")
+  if( ARCH STREQUAL native_cpu )
```

Comment:
I don't think you need this statement anymore - see [these lines in the `else` case above](https://github.com/intel/llvm/pull/17408/files#diff-a74fdaefdc7b79eeee6d67dc1ed4f57389f15864b3e0cd47efc39a2cde137e3cR337-R339)

Reply:
Good spot, thanks. Removed.